qutebrowser/config/configdata.yml

```diff
@@ -1191,9 +1191,14 @@ prompt.radius:
 ## scrolling
 
 scrolling.bar:
-  type: Bool
-  default: false
-  desc: Show a scrollbar.
+  type:
+    name: String
+    valid_values:
+      - always: Always show the scrollbar.
+      - never: Never show the scrollbar.
+      - when_searching: Show the scrollbar when searching for text in the webpage.
+  default: when_searching
+  desc: When to show the scrollbar.
```

Comment:
We should add a note somewhere saying this setting only works on qtwebengine.

I think you can do this by adding:

`  backend: QtWebEngine`

Reply:
I was wondering if/how I could do that! Will also that change the default to `never` when the backend is qtwebkit?

Reply:
Or rather `always`.

Reply:
There are no per-backend default values, but this setting *does* work on QtWebKit (only the `when-searching` value doesn't). I edited the docs accordingly in 2f121d77dfd08eeaaafa8c3f07c20c4f39094f8f.